src/hooks/api/authenticated-query.js

```diff
@@ -134,43 +133,11 @@ const useAuthenticatedMutation = ({
   return useMutation(innerMutationFn, configuration);
 };
 
-const useAuthenticatedQuery = ({ mockData, ...options } = {}) => {
+const useAuthenticatedQuery = ({ ...options } = {}) => {
   if (!!options.req && !!options.queryClient && typeof window === 'undefined') {
     return prefetchAuthenticatedQuery(options);
   }
 
-  const [randomMockData, setRandomMockData] = useState();
```

Comment:
Why remove mockData?

Reply:
See the description in the commit:

> These can be "mocked" in the similar_events table instead so we can also test the actual API integration locally

Reply:
I understand why it was added, but it's not necessary anymore because we're not getting those suggestions from an external API anymore. It's all in the UDB3 backend now (been for a while actually) and easy to add test data with a MySQL QUI app. And then you're testing the actual API call and not a mock.

Reply:
👍🏻

Reply:
@simon-debruijn @geoffreydhuyvetters if you want I can document how to do this exactly? Or is it clear enough?